python/build-requirements.txt

```diff
@@ -9,7 +9,7 @@ pre-commit
 mypy
 
 # ml4ir dependencies
-tensorflow==2.7.4
+tensorflow==2.9.3
```

Comment:
Upgrading because we do not have security approval to use 2.7.x due to known issues.